copyright

```diff
@@ -1068,6 +1100,25 @@ Copyright: Benjamin Jackson (gods.benyamin@outlook.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license)
 
+Files
+ images/*/*ladybug*
```

Comment:
I don't think the copyright is usually done like this?
@endless-sky/art-patrol, this ok?